x-pack/plugins/integration_assistant/public/common/lib/api.ts

```diff
@@ -42,6 +44,16 @@ export interface RequestDeps {
   abortSignal: AbortSignal;
 }
 
+export const runAnalyseLogsGraph = async (
```

Comment:
NIT: naming, not sure if the word "graph" is adequate for this function, maybe we could call this function `runAnalyseLogsContent` or just `runAnalyseLogs`

Reply:
This is a new Graph call actually.

Reply:
you're right, I just checked the server implementation. Ignore this comment then 👍